source/backend/api/Helpers/Healthchecks/PimsMayanHealthCheck.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Pims.Api.Repositories.Mayan;
+
+namespace Pims.Api.Helpers.Healthchecks
+{
+    public class PimsMayanHealthcheck : IHealthCheck
+    {
+        private readonly IEdmsDocumentRepository _documentRepository;
+
+        public PimsMayanHealthcheck(IEdmsDocumentRepository documentRepository)
+        {
+            _documentRepository = documentRepository;
+        }
+
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            try
+            {
+                var documentTypes = await _documentRepository.TryGetDocumentTypesAsync(string.Empty, 1, 1);
```

Comment:
This seemed like a realistic choice from the rest spec, if there is a better choice let me know.